app/js/main.js

```diff
@@ -172,7 +172,6 @@ function setupUpdateSubscription(subscriptions) {
   );
 }
 
-/* eslint-disable */
```

Comment:
I prefer disabling eslint for the entire method. Otherwise the method body has multiple eslint comments which adds noise that might confuse people using this as an example

Reply:
Oh, I just realized disabling eslint altogether breaks the automatic code formatting though, that's unfortunate...

Reply:
I'd rather keep the eslint disabling focused so you don't enable errors for the entire function for a single line that doesn't comply.

Reply:
I just sent you a PR for more focused eslint disabling https://github.com/subscriptions-project/scenic-demo/pull/203